TUTORIAL.md

```diff
@@ -0,0 +1,99 @@
+# Idea
+
+`pylexibank` supports the idea of having [lexibank data sets](https://github.com/lexibank) as Python packages. This allows for an easy management of data sets by facilitating Python's (and `pip`'s) capabilities for installing and managing Python modules.
+
+# Requirements
+
+* Python
+* support for virtual environments (i.e. a working `virtualenv` command)
+* `pip`
+* `git`
+
+# Installation
+
+Start by preparing your working environment:
+```
+mkdir lexibank-working
+cd lexibank-working
+virtualenv venv
+source venv/bin/activate
+```
+
+Your command prompt should now show `(venv)` before all your commands; this means that the virtual environment is ready and in use (if, at any time, you'd like to leave the virtual environment, just type `deactivate`).
+
+Let's continue by installing the necessary requirements (everything still happens in the `lexibank-working` directory with the previously activated virtual environment):
+
+```
+pip install https://github.com/lexibank/pylexibank/archive/master.zip
```

Comment:
`pylexibank` has been released on pypi now, so installation can be done via `pip install pylexibank`.